internal/cmd/gocdk/testdata/biome-help.ct

```diff
@@ -1,61 +1,73 @@
 $ gocdk help biome
-TODO more about biomes
+Biomes are target environments. Each biome has a distinct deployment target,
+and maintains its own set of Cloud resources.
+
+A typical set of biomes is "dev", "staging", "prod".
 
 Usage:
   gocdk biome [command]
 
 Available Commands:
-  add         TODO Add BIOME_NAME
-  apply       TODO Apply Terraform for BIOME
-  list        TODO list
+  add         Add a new biome
+  apply       Apply any changes required for the biome's resource configuration
+  list        Print the list of existing biomes
 
 Flags:
   -h, --help   help for biome
 
 Use "gocdk biome [command] --help" for more information about a command.
 
 $ gocdk biome -h
-TODO more about biomes
+Biomes are target environments. Each biome has a distinct deployment target,
+and maintains its own set of Cloud resources.
+
+A typical set of biomes is "dev", "staging", "prod".
 
 Usage:
   gocdk biome [command]
 
 Available Commands:
-  add         TODO Add BIOME_NAME
-  apply       TODO Apply Terraform for BIOME
-  list        TODO list
+  add         Add a new biome
+  apply       Apply any changes required for the biome's resource configuration
```

Comment:
I'm concerned not everyone will understand that applying configuration changes will mean provisioning cloud resources, can we add something like "(e.g. creating a bucket)"

Reply:
Sure! Done.